benchmark/paddle/DQN/replay_memory.py

```diff
@@ -0,0 +1,46 @@
+#   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
Can we reuse the rpm in `parl.utils`?
https://github.com/PaddlePaddle/PARL/blob/develop/parl/utils/replay_memory.py